impl/src/test/groovy/io/jsonwebtoken/impl/DefaultJwtParserBuilderTest.groovy

```diff
@@ -92,4 +96,25 @@ class DefaultJwtParserBuilderTest {
             assertEquals DefaultJwtParserBuilder.MAX_CLOCK_SKEW_ILLEGAL_MSG, expected.message
         }
     }
+
+    @Test
+    void testDefaultDecoder() {
+        JwtParser parser = new DefaultJwtParserBuilder().build()
+        assertThat parser.jwtParser.deserializer, CoreMatchers.instanceOf(JwtDeserializer)
+
+        // TODO: When the ImmutableJwtParser replaces the default implementation this test will need updating, something like:
+        // assertThat parser.deserializer, CoreMatchers.instanceOf(JwtDeserializer)
+    }
+
+    @Test
+    void testUserSetDecoderWrapsImplementation() {
```

Comment:
Really minor, but unlike the test methods above which actually set a `Decoder`, this one sets a `Deserializer`
(the method name might also be misleading because not the user deserializer wraps, but it is wrapped)
```suggestion
    void testUserSetDeserializerWrapped() {
```